.gitignore

```diff
@@ -13,7 +13,6 @@ debug.log
 node_modules/
 *.code-workspace
 node_modules
-.vscode/
```

Comment:
Do we potentially want to restrict this to just `.vscode/settings.json` as I'm not sure if we necessarily want common settings

Reply:
From my past experience, `.vscode/settings.json` has always been used for shared workspace settings, which overwrites any user configuration. I'm happy to scope down the `.gitignore` to just `launch.json` while keeping `settings.json` customizable, if that's not been used so far.

Reply:
Yeah thats what I was thinking for now. Just ignore everything that **isn't** `launch.json`. At least for this PR. Not against adding a common `settings.json` if we agree on a sensible list though.

Reply:
Just pushed a change to fix this, thanks!